src/AdminConsole/Components/Pages/App/Settings/AuthenticationConfiguration/AuthenticationConfigurationForm.razor

```diff
@@ -0,0 +1,56 @@
+@using Fido2NetLib.Objects
+@inherits BaseRazorComponent
+
+@inject ILogger<AuthenticationConfigurationForm> Logger
+@inject IScopedPasswordlessClient Passwordless
+@inject ICurrentContext CurrentContext
+
+<EditForm Model="FormModel" OnValidSubmit="SubmitAsync" FormName="@FormName" class="space-y-6">
+    <DataAnnotationsValidator/>
+    <ValidationSummary/>
+    <div class="sm:w-full sm:max-w-md">
+        <label class="block text-sm font-medium leading-6 text-gray-900 my-2" for="Purpose">Purpose:</label>
+        <InputText @bind-Value="@FormModel!.Purpose" id="Purpose" disabled="@IsPurposeReadOnly" class="text-input"/>
+        <label class="block text-sm font-medium leading-6 text-gray-900 my-2" for="UserVerification">User Verification:</label>
+        <InputSelect id="UserVerification" @bind-Value="@FormModel.UserVerificationRequirement" TValue="UserVerificationRequirement" class="form-select">
+            @foreach (var uv in Enum.GetValues(typeof(UserVerificationRequirement)))

```

Comment:
Nit: if you use `Enum.GetValues<UserVerificationRequirement>()`, you'll get typed outputs. Doesn't matter much here if you're only calling `ToString()` on them anyway.